resources/lang/vi/object.php

```diff
@@ -7,5 +7,5 @@
     'list' => 'Danh sách :object',
     'management' => 'Quản lý :object',
     'reset' => 'Cấp lại :object',
-    'trashed' => ':object đã tạm khóa',
+    'trashed' => ':object Đã xóa',
```

Comment:
```suggestion
    'trashed' => ':object đã xóa',
```
giữa câu ko viết hoa.